cloudbuild.yaml

```diff
@@ -31,19 +31,49 @@ steps:
   args: ['tag', 'bazel/java:java8', 'gcr.io/$PROJECT_ID/java:latest']
 
 - name: gcr.io/cloud-builders/bazel
-  args: ['run', '//java:debug']
+  args: ['run', '//java:java8']
+- name: gcr.io/cloud-builders/docker
```

Comment:
I'm not sure how this whole section works, but do we need to rerun the bazel build of the image for each tag of the same build?

looks like we have
```
- name: gcr.io/cloud-buidlers/bazel
  args: ['run', '//java:java8']
```
on line 28 and line 33

Reply:
Ah, good catch. I think we can just tag it multiple times. Will updated it.

Note, it's possible that we will remove this `cloudbuild.yaml` at any point (#286).